astropy/visualization/wcsaxes/core.py

```diff
@@ -96,19 +108,28 @@ class WCSAxes(Axes):
         :class:`~astropy.visualization.wcsaxes.frame.RectangularFrame`
     """
 
-    def __init__(self, fig, rect, wcs=None, transform=None, coord_meta=None,
-                 transData=None, slices=None, frame_class=None,
-                 **kwargs):
-        """
-        """
+    def __init__(
+        self,
+        fig,
+        rect,
+        wcs=None,
+        transform=None,
+        coord_meta=None,
+        transData=None,
+        slices=None,
+        frame_class=None,
+        **kwargs,
+    ):
+        """ """
```

Comment:
We are still discussing whether or not to adopt black formatting for astropy (see https://github.com/astropy/astropy-APEs/pull/76) - since this hasn't been approved yet (and may not be), could you undo any formatting changes for now? (alternatively you could always wait a few weeks to see if the APE is approved, if undoing the formatting changes would be too much work).

Reply:
OK, done.